spec/features/admin_creates_quiz_spec.rb

```diff
@@ -106,6 +106,59 @@
       expect_to_see "Answers - at least 1 must be correct."
     end
   end
+
+  context "client side validations" do
+    scenario "check on quiz", js: true, driver: :selenium do
+      fill_in "Title", with: ""
+      click_away
+      expect_to_see "This value is required."
+      fill_in "Title", with: "Some title"
+      click_away
+      expect_to_not_see "This value is required."
+
+      fill_in "Description", with: ""
+      click_away
+      expect_to_see "This value is required."
+      fill_in "Description", with: "Some description"
+      click_away
+      expect_to_not_see "This value is required."
+    end
+
+    scenario "check on question", js: true, driver: :selenium do
+      fill_in_valid_quiz_attributes
+      add_question(1, with: "", points: 3) do |question|
```

Comment:
Unused block argument - `question`. You can omit the argument if you don't care about it.